lib/ui/painting/picture.cc

```diff
@@ -81,8 +82,7 @@ Dart_Handle Picture::RasterizeToImage(sk_sp<SkPicture> picture,
   }
 
   auto* dart_state = UIDartState::Current();
-  tonic::DartPersistentValue* image_callback =
-      new tonic::DartPersistentValue(dart_state, raw_image_callback);
+  auto image_callback = std::make_shared<tonic::DartPersistentValue>(dart_state, raw_image_callback);
```

Comment:
I think this is good.  FWIW I was thinking you could use `unique_ptr`.  Then when you capture it in the lambda you can use `std::move`.  It's slightly more efficient.

```c++
auto foo = std::make_unique<Foo>();
auto ui_task = fml::MakeCopyable([bar = std::move(foo)]() mutable {
});
```

But I don't think that would have worked because ui_task is copied.  There would have to be further refactoring to make that work.